pickle/src/com/treode/pickle/PicklerRegistry.scala

```diff
@@ -14,100 +14,269 @@
  * limitations under the License.
  */
 
-package com.treode.pickle
+ package com.treode.pickle
```

Comment:
I tend to be more lenient with style than other projects, but this is definitely outside the bounds of acceptable. Indentation changes oddly throughout the file. Clean that up please.